server/apps/acсount/models.py

```diff
@@ -0,0 +1,200 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.db.models import TextChoices, QuerySet
+from PIL import Image
+from django.core.exceptions import ValidationError
+from django.core.validators import FileExtensionValidator
+from typing import Any
+from django.utils import timezone
+    
+
+def validator_avatar_weight(file: Any) -> None:
+    """Валидатор размера файла для изображений."""
+    limit = 5 * 1024 * 1024
+    if file.size > limit:
+        raise ValidationError('нюхай бебру')
+
+
+class User(AbstractUser):
+    class GENDERS(TextChoices):
+        MALE = 'male', 'Male'
+        FEMALE = 'female', 'Female'
+
+    gender = models.CharField(
+        max_length=6,
+        choices=GENDERS.choices,
+        default=GENDERS.MALE,
+        )
+    avatar = models.ImageField(
+        verbose_name='Avatar',
+        upload_to='media/',
```

Comment:
If u trying to upload some image, u will catch an error.

Before using `ImageField`, you need to define MEDIA_ROOT in the `settings` file.

Source:
https://docs.djangoproject.com/en/5.1/ref/settings/#media-root
https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.FileField.storage